app/questionnaire/questionnaire_schema.py

```diff
@@ -62,7 +62,7 @@ def __init__(
     ):
         self._parent_id_map: dict[str, str] = {}
         self._list_name_to_section_map: dict[str, list[str]] = {}
```

Comment:
I suggest updating this existing one to something like `self._section_dependenices_by_list_name`.  Open to other suggestions.

Reply:
Updated 👍